buildenv/jenkins/jobs/infrastructure/artifactoryCheck.groovy

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+def LABEL = (params.LABEL != "") ? params.LABEL : "artifactory"
+def WARNING_THRESHOLD = (params.WARNING_THRESHOLD != "") ? params.WARNING_THRESHOLD : "75"
+def DANGER_THRESHOLD = (params.DANGER_THRESHOLD != "") ? params.DANGER_THRESHOLD : "90"
+def SLACK_CHANNEL = (params.SLACK_CHANNEL != "") ? params.SLACK_CHANNEL : "#jenkins-sandbox"
+HIGH_ROOT_DISK_PERCENTAGE = 0
+HIGH_ARTIFACTORY_DISK_PERCENTAGE = 0
+
+def artifactoryNodes = nodesByLabel label:"${LABEL}", offline: false
+
+timestamps {
+	echo "Number of Nodes that match labels: " + artifactoryNodes.size()
+	for (sNode in artifactoryNodes) {
+		node("${sNode}") {
+			stage("${sNode} Artifactory data collection\n") {
+				ROOT_DISK_DETAILS = sh (label: 'Get / disk details', returnStdout: true, script: "df / -h | tail -1 | awk '{print \$3 \" / \" \$2}'").trim()
+				ROOT_DISK_PERCENTAGE = sh (label: 'Get / disk percentage', returnStdout: true, script: "df / -h | tail -1 | awk '{print \$5}'").trim()
+				ARTIFACTORY_DISK_DETAILS = sh (label: 'Get Artifactory disk details', returnStdout: true, script: "df -h | grep jenkins | awk '{print \$3 \" / \" \$2}'").trim()
+				ARTIFACTORY_DISK_PERCENTAGE = sh (label: 'Get Artifactory disk percentage', returnStdout: true, script: "df -h | grep jenkins | awk '{print \$5}'").trim()
```

Comment:
Maybe a bit pedantic but if we're using ALLCAPS for globals then I think these should be `def`'d and in camelCase.

Reply:
Updated in the latest update.